system-addon/lib/TelemetryFeed.jsm

```diff
@@ -201,8 +210,10 @@ this.TelemetryFeed = class TelemetryFeed {
     Services.obs.removeObserver(this.browserOpenNewtabStart,
       "browser-open-newtab-start");
 
-    this.telemetrySender.uninit();
-    this.telemetrySender = null;
+    // Only uninit if the getter has initialized it
+    if (Object.prototype.hasOwnProperty.call(this, "telemetrySender")) {
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/hasOwnProperty#Description
"unlike the `in` operator, this method does not check down the object's prototype chain."

Maybe we were thinking of the behavior of `for … in` skipping over non-enumerable https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...in